web/json_value_test.go

```diff
@@ -93,7 +93,7 @@ func TestConvertJSONAttributeValue(t *testing.T) {
 				ExternalId: "a",
 				Type:       framework.AttributeTypeDateTime,
 			},
-			valueJSON: `"1706041056"`,
+			valueJSON: `1706041056`,
```

Comment:
Add a test for parsing a number with some digits after the decimal e.g. `3.1415`. That would test the condition `float64(int64(v)) != v`.